finality-provider/config/config.go

```diff
@@ -55,16 +61,16 @@ type Config struct {
 	// ChainType and ChainID (if any) of the chain config identify a consumer chain
 	ChainType                   string        `long:"chaintype" description:"the type of the consumer chain" choice:"babylon"`
 	NumPubRand                  uint32        `long:"numPubRand" description:"The number of Schnorr public randomness for each commitment"`
-	NumPubRandMax               uint32        `long:"numpubrandmax" description:"The upper bound of the number of Schnorr public randomness for each commitment"`
-	TimestampingDelayBlocks     uint32        `long:"timestampingdelayblocks" description:"The delay, measured in blocks, between a randomness commit submission and the randomness is BTC-timestamped"`
 	MaxSubmissionRetries        uint32        `long:"maxsubmissionretries" description:"The maximum number of retries to submit finality signature or public randomness"`
 	EOTSManagerAddress          string        `long:"eotsmanageraddress" description:"The address of the remote EOTS manager; Empty if the EOTS manager is running locally"`
 	HMACKey                     string        `long:"hmackey" description:"The HMAC key for authentication with EOTSD. If not provided, will use HMAC_KEY environment variable."`
 	BatchSubmissionSize         uint32        `long:"batchsubmissionsize" description:"The size of a batch in one submission"`
-	RandomnessCommitInterval    time.Duration `long:"randomnesscommitinterval" description:"The interval between each attempt to commit public randomness"`
 	SubmissionRetryInterval     time.Duration `long:"submissionretryinterval" description:"The interval between each attempt to submit finality signature or public randomness after a failure"`
 	SignatureSubmissionInterval time.Duration `long:"signaturesubmissioninterval" description:"The interval between each finality signature(s) submission"`
 
+	// not configurable in config file
+	TimestampingDelayBlocks uint32
```

Comment:
Why add it here instead of loading the default from the package directly as you do for `RandCommitInterval, `MinNumPubRand` and so on?

Reply:
This is because we need to set a small value for e2e test to pass quickly

Reply:
hmm given that this poses some problems for:
- e2e tests
- switching between testnet / mainnet
- potential problems for phase-3 (which we went around by making pr to `release/v1.x`)

Maybe lets leave it as config value with default set to mainnet value ? and in future pr's introduce validation based on `W` from babylon node or even set it based on `W` from Babylon node

Reply:
No problem with me. I can add the value for testnet in the commends. Do we have a consensus on the default value for both mainnet and test https://github.com/babylonlabs-io/finality-provider/pull/418#discussion_r2020852928?